docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -200,16 +212,22 @@ To learn more about installing, configuring, and running {es}, read the
 https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html[{es} Reference].
 
 [float]
-==== Make sure {es} is up and running
-
+==== Make sure that {es} is up and running
 
-To test that the {es} daemon is up and running, try sending an HTTP GET
-request on port 9200.
+Open a new terminal and verify that you can connect to your {es} cluster by
+making an authenticated call. Enter the password for the `elastic` user when
+prompted:
 
 [source,shell]
-----------------------------------------------------------------------
-curl http://127.0.0.1:9200
-----------------------------------------------------------------------
+----
+curl --cacert config/tls_auto_config_<timestamp>/http_ca.crt \
+-u elastic https://localhost:9200 <1>
+----
+// NOTCONSOLE
+<1> Ensure that you use `https` in your call, or the request will fail.
```

Comment:
I've seen in other places over the last PRs and it troubled me more and more so now reached my "need-to-comment" threshold :) 

The <1> callout makes sense in the context of the diff of this PR but not necessarily when you read the docs as a user. The callout is true, but it's also _too(?)_ obvious. As in we could also say, `Ensure that you use `9200` as the port in your call, or the request will fail.`

WDYT ?

Reply:
@lockewritesdocs  TBC, I'm not suggesting we add "Ensure that you use 9200 as the port in your call, or the request will fail." here. It was just a comment about another thing that while true, is (IMO) too obvious to point out and I doubt the usefulness. My suggestion is to remove the existing callout.

Reply:
I thought about this a bit, and I agree that we can remove this callout. If we get feedback that users get stuck, we can always add something.